src/test/kotlin/com/noumenadigital/npl/cli/DeployCommandIT.kt

```diff
@@ -480,71 +465,32 @@ class DeployCommandIT :
 
         context("command validation errors") {
             test("missing target parameter") {
+                val testDirPath =
+                    getTestResourcesPath(listOf("deploy-success", "main")).toAbsolutePath().toString()
+
                 runCommand(
-                    commands = listOf("deploy"),
+                    commands = listOf("deploy", "--sourceDir=$testDirPath"),
                 ) {
                     process.waitFor(5, TimeUnit.SECONDS)
 
                     val expectedOutput =
-                        """
-                    Missing required parameter: target
-                    Usage: deploy <target> <directory> [--clear]
-
-                    Arguments:
-                      target           Named target from deploy.yml to deploy to
-                      directory        Directory containing NPL sources.
-                                       IMPORTANT: The directory must contain a valid NPL source structure, including
-                                       migrations. E.g.:
-                                        main
-                                        ├── npl-1.0
-                                        │   └── processes
-                                        │       └── demo.npl
-                                        └── yaml
-                                            └── migration.yml
-
-                    Options:
-                      --clear          Clear application contents before deployment
-
-                    Configuration is read from .npl/deploy.yml in the current directory
-                    or the user's home directory (~/.npl/deploy.yml).
-                """.normalize()
+                        "Missing required parameter: --target=<name> (or use --dev for local defaults)\n" +
+                            DeployCommand.USAGE_STRING
```

Comment:
I  know it's depending on prod classes, but in this case I see no problem with it. Makes it much more maintainable with a bit of DRY.